README.md

```diff
@@ -126,6 +126,23 @@ assert(HARDENED_FLAG === 0x80000000);
 
 ```
 
+#### Bytes
+
+```js
+
+import { bytes } from '@bpanel/bpanel-utils';
```

Comment:
this would need to be `import { helpers }` and then `helpers.isU8` now right?

Reply:
Just fixed that, good call